library/libraries.js

```diff
@@ -993,8 +993,8 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
         icon: 'vue',
         url: 'http://vuejs.org/',
         test: function(win) {
-            if (win.Vue && win.Vue.compiler) {
-                return { version: 'N/A' };
+            if (win.Vue && win.Vue.version) {
```

Comment:
While I understand that it is part of the API, I prefer not to use the `.version` property check for any library as a way of determining if a library is on the page or not.  Something more specific like [`Vue.nextTick`](http://vuejs.org/v2/api/#Vue-nextTick) would be preferred.